nuimo.py

```diff
@@ -307,7 +316,7 @@ def write_matrix(self, matrix, timeout, brightness=1.0):
         led_data = bytearray(matrix_bytes_list)
         led_uuid = self.characteristics_by_name['LED_MATRIX']['uuid']
         led_handle = self.value_handle(led_uuid)
-        self.write_by_handle(led_handle, bytes(led_data))
+        self.write_by_handle_async(led_handle, bytes(led_data), response)
```

Comment:
It's mandatory for async write, i don't use it, but it would a way to detect that the write indeed happened, i though that maybe we should keep track of that and only send new write instructions if the queue is cleared, but i didn't try doing that yet. I'm still unsure about the perfect strategy, because it's not uncommon that write instructions stops working, either sync or async (but the other instructions keep working if the write calls were async). I don't see such issues with your android nuimo app and e.g soundcloud integration, so something must be done differently in there.